urbanairship-chat-react-native/ios/AirshipChatModule.m

```diff
@@ -0,0 +1,66 @@
+#import "AirshipChatModule.h"
+#import "ConversationUpdatedDelegate.h"
+#import "OpenChatDelegate.h"
+@import Airship;
+@implementation AirshipChatModule
+
+bool useCustomChatUI;
+
+RCT_EXPORT_MODULE()
+
+- (instancetype)init{
+    self = [super init];
+    if (self) {
+        [UAirshipChat shared].openChatDelegate = [OpenChatDelegate shared];
```

Comment:
since there is no returning true/false if its handled, you need to only add this listener if useCustomUI is true.

probably call `updateOpenChatDelegate` here. In that method check the property and add/remove the listener depending on the value. When useCustomUI value changes, call `updateOpenChatDelegate` again